lib/pages/new_group/widget/contacts_selection_list.dart

```diff
@@ -12,6 +14,7 @@ class ContactsSelectionList extends StatelessWidget {
   final Function() onSelectedContact;
   final List<String> disabledContactIds;
   final TextEditingController textEditingController;
+  final ContactsManager contactsManager;
```

Comment:
it should be better for singleton instance
```suggestion
  final ContactsManager contactsManager = getIt<ContactsManager>();
```

Reply:
No need, I'll remove it